### PR TITLE
refactor: removed unused css files

### PR DESCRIPTION
**Pull request by nayyyhaa**
- removed unused CSS files.
- aligned components properly.

### CODE REVIEW

File: docs/alert/index.html [hunk 1]
```diff
@@ -142,7 +142,7 @@ <h3 class="title">Code Snippets</h3>
 
           <!-- Outlined Alerts -->
           <h2 class="title h2 m-v-2 colored-bg">Outlined alerts</h2>
-          <div class="container colored-container col-flex">
+          <div class="container colored-container m-v-4 col-flex">
```
Comment (shahbazibnafzal) [inline docs/alert/index.html:145]: I think naming for margin and padding classes could be better, rest of everything is good.

Comment (nayyyhaa) [reply docs/alert/index.html:145]: inspired from tailwind CSS 😅 but, sure will try to rename it.

**Review by Dev-AkshatGupta — APPROVED** [review]
My first reaction is Woahh!! How it can be so good 
This is really amazing Neha One of the coolest UI have encountered I would say

**Review by ParthParolekar — APPROVED** [review]
The code looks clean. Great work :+1:

**Comment by nayyyhaa** [comment]
Thanks all for reviewing. Merging the changes.

**Pull request merged**